src/main.cpp

```diff
@@ -262,6 +266,14 @@ int mainEmulatorHLE()
 	PPCCore_init();
 	// log Cemu startup info
 	infoLog_cemuStartup();
+	// enable gamemode (Linux-only library for standby prevention and some OS optimization)
+	#if BOOST_OS_LINUX
+		if (GetConfig().gamemode && gamemode_request_start() < 0)
+			cemuLog_force("Failed to enable gamemode, it may not be installed");
+			// TODO: disable gamemode when only the game, not Cemu, is closed (a feature not yet implemented)
+			// currently unnecessary because this will happen automatically when Cemu closes
+			// gamemode_request_end();
+	#endif
```

Comment:
mainEmulatorHLE is a function that runs when cemu starts and in the settings you only start gamemode if a game is running. Should probably move this somewhere like MainWindow::FileLoad so it only runs when a game is started.